### ISSUE TITLE
Stripping spaces from string in directive

### ISSUE THREAD

**Issue by howdu** [Stillat/blade-parser-typescript#77]
```blade
@props([
   'product',
   'title' => $product->name . ' - ' . $product->category,
])
```

Becomes
```blade
@props([
   'product',
   'title' => $product->name . '-' . $product->category,
])
```

**Comment by JohnathonKoster** [comment]
Would you be able to share your prettier config as well as the formatting config (if applicable)? Thanks!

**Comment by howdu** [comment]
Sure no problem it's:
```json
{
  "semi": false,
  "endOfLine": "lf",
  "singleQuote": true,
  "tailwindConfig": "./tailwind.config.js",
  "trailingComma": "all",
  "singleAttributePerLine": true,
  "plugins": [
    "./node_modules/prettier-plugin-blade/",
    "./node_modules/prettier-plugin-tailwindcss/"
  ],
  "overrides": [
    {
      "files": ["*.blade.php"],
      "options": {
        "parser": "blade"
      }
    }
  ]
}
```

**Comment by JohnathonKoster** [comment]
Thank you very much 🙂

I've identified the issue, and have released `1.6.12` to address this; with additional test coverage going forward.
